tests/integration/functional/zen_stores/test_zen_store.py

```diff
@@ -377,6 +381,173 @@ def test_deleting_default_workspace_fails():
 # '-------'
 
 
+class TestAdminUser:
```

Comment:
Thanks for the hint. I might need to explain a bit why I'm using it here and there: the main thing is the fact that test files are big and often time there is no such a big reason to run all of them locally, rather run mostly impacted ones by recent changes. How class could help here? Rather simple: by calling `pytest tests/.../test_file.py::TestClass` I get only tests of `TestClass` executed, rather than all tests in `test_file.py`. So, if you don't mind, I prefer to keep it as is and group tests in classes going forward.
If there is any specific downsides of this, except the fact, that it is not perfectly aligned with what was developed already - would love to discuss it further.

Reply:
That's a very good point, I hadn't thought of it. Thanks for explaining it.